### PR TITLE
Bump libovsdb to 0.5.0

### PR DESCRIPTION
**Pull request by masap**
Bump libovsdb to 0.5.0.
And bump Go to 1.16 because libovsdb 0.4.0 requires it.

release-note
```release-note
Bump Go to 1.16
Bump libovsdb to 0.4.0
Bump libovsdb to 0.5.0
```

Signed-off-by: Masashi Honma masashi.honma@gmail.com

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -35,6 +35,8 @@ $(BASE): ; $(info  setting GOPATH...)
 GOLINT = $(GOBIN)/golint
 $(GOBIN)/golint: | $(BASE) ; $(info  building golint...)
 	$Q go get -u golang.org/x/lint/golint
+	# golint installation modifies go.mod and it causes golint failure so run mod tidy here
```
Comment (phoracek) [inline Makefile:38]: Can't this be avoided by calling `make dep`?

File: pkg/ovsdb/ovsdb.go [hunk 2]
```diff
@@ -30,26 +32,85 @@ const (
...
 
-// Bridge Model of the Bridge table
+type (
+	// BridgeFailMode defines failure mode
+	BridgeFailMode = string
+	// BridgeProtocols defines OpenFlow Protocol version
+	BridgeProtocols = string
+)
+
+const (
```
Comment (phoracek) [inline pkg/ovsdb/ovsdb.go:42]: Please keep only values that are used in the project

File: pkg/ovsdb/ovsdb.go [hunk 3]
```diff
@@ -30,26 +32,85 @@ const (
...
+
+// Bridge defines an object in Bridge table
 type Bridge struct {
-	UUID        string            `ovs:"_uuid"`
-	Name        string            `ovs:"name"`
-	OtherConfig map[string]string `ovs:"other_config"`
-	ExternalIds map[string]string `ovs:"external_ids"`
-	Ports       []string          `ovs:"ports"`
-	Status      map[string]string `ovs:"status"`
+	UUID                string            `ovsdb:"_uuid"`
```
Comment (phoracek) [inline pkg/ovsdb/ovsdb.go:63]: We don't  use most of these. Could we clean it up, so we don't bloat the module too much? This project has the benefit of not being a library but a focused application. We should only keep code that is needed for the project.

**Review by phoracek — CHANGES_REQUESTED** [review]

File: Makefile (hunk 1 above)
Comment (masap) [reply Makefile:38]: No. Running ```make dep``` before ```make lint``` does not change the result.

File: pkg/ovsdb/ovsdb.go (hunk 2 above)
Comment (masap) [reply pkg/ovsdb/ovsdb.go]: Ok, fixed.

File: pkg/ovsdb/ovsdb.go (hunk 3 above)
Comment (masap) [reply pkg/ovsdb/ovsdb.go]: Ok, fixed.

**Review by phoracek — APPROVED** [review]
I'm surprised we don't need any attributes in the schema, but it seems to work anyway.

Thanks

/lgtm
/approve

**Pull request merged**